string[] joint_names
trajectory_msgs/JointTrajectoryPoint goalPoint
---
